deploy/dynamic.yml

```diff
@@ -1,72 +0,0 @@
-http:
```

Comment:
It's ok to remove these files, but should also update the deploy section of the readme: https://github.com/nethesis/fpp/blob/master/README.md#build-and-deploy